File: keiner_lindarte_aro/instructor_henry/mis_programas/practicas/java/practicas3/estratos.java
package ADSO2928707.portafolio_aprendices.keiner_lindarte_aro.instructor_henry.mis_programas.practicas.java.practicas3;

import java.util.Scanner;

public class estratos {

    public static void main(String[] args) {
        
        try (Scanner input = new Scanner (System.in)){

            System.out.println("Cuantas personas se van a encuestar?");
            int cuantos = input.nextInt();
          
            int state1 = 0;
            int state2 = 0;
            int state3 = 0;
            int state4 = 0;
            int state5 = 0;
            int state6 = 0;

            for (int vueltas = 1; vueltas <= cuantos; vueltas++) {

                System.out.println("----------------------------------------------------------");
            
                System.out.println("¿De que tipo de estrato es?");
                int state = input.nextInt();
            
                if (state == 1){
                  state1 = state1 + 1;
                } else if (state == 2){
                  state2 = state2 + 1;
                  } else if (state == 3){
                  state3 = state3 + 1;
                  } else if (state == 4){
                  state4 = state4 + 1;
                  } else if (state == 5){
                  state5 = state5 + 1;
                  } else if (state == 6){
                  state6 = state6 + 1;
                  }
            
                System.out.println("----------------------------------------------------------");
            
                System.out.println("Su informacion ha sido registrada correctamente.");
            
                System.out.println("----------------------------------------------------------");
            
              }
            
              System.out.println("----------------------------------------------------------");
            
              System.out.println("CLASIFICACION DE LA ENCUESTA REALIZADA:");
            
              System.out.println("----------------------------------------------------------");
            
              System.out.println("Cantidad de personas de estrato 1:");
              System.out.println(state1);
            
              System.out.println("Cantidad de personas de estrato 2:");
              System.out.println(state2);
            
              System.out.println("Cantidad de personas de estrato 3:");
              System.out.println(state3);
            
              System.out.println("Cantidad de personas de estrato 4:");
              System.out.println(state4);
            
              System.out.println("Cantidad de personas de estrato 5:");
              System.out.println(state5);
            
              System.out.println("Cantidad de personas de estrato 6:");
              System.out.println(state6);
            
              System.out.println("----------------------------------------------------------");



        }


    }
    
}

File: keiner_lindarte_aro/instructor_henry/mis_programas/practicas/python/practicas3/estratos.py

print( "Cuantas personas se van a encuestar?")
cuantos = int(input())

state1 = 0
state2 = 0
state3 = 0
state4 = 0
state5 = 0
state6 = 0

for vueltas in range(1,cuantos + 1):

    print( "----------------------------------------------------------")

    print( "¿De que tipo de estrato es?")
    state = int(input())

    if state == 1:
        state1 = state1 + 1
    if state == 2:
        state2 = state2 + 1
    if state == 3:
        state3 = state3 + 1
    if state == 4:
        state4 = state4 + 1
    if state == 5:
        state5 = state5 + 1
    if state == 6:
        state6 = state6 + 1
    
print( "----------------------------------------------------------")

print( "Su informacion ha sido registrada correctamente.")

print( "----------------------------------------------------------")

print( "----------------------------------------------------------")

print( "CLASIFICACION DE LA ENCUESTA REALIZADA:")

print( "----------------------------------------------------------")

print( "Cantidad de personas de estrato 1:")
print( state1)

print( "Cantidad de personas de estrato 2:")
print( state2)

print( "Cantidad de personas de estrato 3:")
print( state3)

print( "Cantidad de personas de estrato 4:")
print( state4)

print( "Cantidad de personas de estrato 5:")
print( state5)

print( "Cantidad de personas de estrato 6:")
print( state6)

print( "----------------------------------------------------------")


